components/reusable-components/_Types/index.ts

```diff
@@ -44,3 +44,9 @@ export interface BlogDetailsData {
   publishDate: string;
   tags: string[];
 }
+
+export interface User {
+  name: string;
+  email: string;
+  avatar: string;
+}
```

Comment:
Always consult with the api documentaton before creating interfaces.
Here the User should have: firstName, lastName, email, status, image.
Status will be an enum, consisting of : active, banned, deleted